.github/workflows/cla-assistant.yaml

```diff
@@ -0,0 +1,47 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened, closed, synchronize]
+# explicitly configure permissions, in case your GITHUB_TOKEN workflow permissions are set to read-only in repository settings
+permissions:
+  actions: read
+  contents: read # this can be 'read' if the signatures are in remote repository
+  pull-requests: write
+jobs:
+  CLAAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - id: fetch-token
+        name: Fetch installation token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.CLA_ASSISTANT_APP_ID }}
+          private-key: ${{ secrets.CLA_ASSISTANT_APP_PRIVATE_KEY }}
+          repositories: cla-signatures
+      - id: get-membership
+        name: "Determine gravitational membership"
+        env:
+          GITHUB_TOKEN: ${{ steps.fetch-token.outputs.token }}
+          USER: ${{ github.actor }}
+        continue-on-error: true
+        # Use the "Check Membership" API for user
+        # If the user is a member of the org expect a 204 HTTP Status Code
+        # If the above is success the process will exit 0
+        run: |
+          gh api "orgs/gravitational/members/${USER}"
+      - name: "CLA Assistant"
+        if: steps.get-membership.outcome != 'success' && ((github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target')
+        uses: contributor-assistant/github-action@v2.3.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PERSONAL_ACCESS_TOKEN: ${{ steps.fetch-token.outputs.token }}
```

Comment:
Will this work with PATs disabled across the org?

Reply:
I'm not 100% sure but probably. It isn't actually a PAT but a different token type granting [app installations authentication for API](https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/authenticating-as-a-github-app-installation).

This works similar to our reviewers bot and already functions under our org. Is this already disabled? If so then yes it currently works.

Reply:
Yep it's been disabled for months